validations/transfer.test.ts

```diff
@@ -0,0 +1,155 @@
+import { expect, use } from 'chai'
+import { ethers } from 'ethers'
+import path from 'path'
+import { randomUUID } from 'crypto'
+import { AddKycParams, FiatConnectClient } from '@fiatconnect/fiatconnect-sdk'
+import {
+  KycSchema,
+  TransferStatus,
+  Network,
+} from '@fiatconnect/fiatconnect-types'
+import { config } from '../src/config'
+import { chaiPlugin } from 'api-contract-validator'
+import { MOCK_QUOTE } from '../src/mock-data/quote'
+import { MOCK_KYC } from '../src/mock-data/kyc'
+import { MOCK_FIAT_ACCOUNTS } from '../src/mock-data/fiat-account'
+
+const apiDefinitionsPath = path.join(config.openapiSpec)
+use(chaiPlugin({ apiDefinitionsPath }))
+
+describe('/transfer', () => {
+  const mockAccountData =
+    MOCK_FIAT_ACCOUNTS[
+    config.fiatAccountMock as keyof typeof MOCK_FIAT_ACCOUNTS
+  ]
+
+  const mockKYCInfo: AddKycParams<KycSchema> =
+    MOCK_KYC[config.kycMock as keyof typeof MOCK_KYC]
+
+  if (config.quoteInMock) {
```

Comment:
sorry I didn't notice this earlier, but these tests are almost identical except for
- what transfer statuses are acceptable
- what method on the FiatConnect client should be used to create the transfer (transferIn vs transferOut)
- which quote to use (config.quoteInMock or config.quoteOutMock)

which means you could make the tests more DRY with `it.each`. You can refer to auth.test.ts for an example. You could even make the switching on/off of tests based on what's in the config a little cleaner that way, for example:
```
const transferTestCases = [] // should add an explicit type here, leaving that as an exercise for the reader ;)
if (config.quoteInMock) {
  transferTestCases.push({quoteParams: config.quoteInMock, ...otherStuff})
}
if (config.quoteOutMock) {
  // TODO
}
it.each(transferTestCases)(...)
```